src/Plugin/pathauto/AliasType/EntityAliasTypeBase.php

```diff
@@ -267,12 +267,12 @@ protected function bulkUpdate(array $ids, array $options = array()) {
     $options += array('message' => FALSE);
 
     $entities = $this->entityManager->getStorage($this->getEntityTypeId())->loadMultiple($ids);
-    foreach ($entities as $node) {
-      \Drupal::service('pathauto.manager')->updateAlias($node, 'bulkupdate', $options);
+    foreach ($entities as $entity) {
+      \Drupal::service('pathauto.manager')->updateAlias($entity, 'bulkupdate', $options);
     }
 
     if (!empty($options['message'])) {
-      drupal_set_message(\Drupal::translation()->formatPlural(count($ids), 'Updated URL alias for 1 node.', 'Updated URL aliases for @count nodes.'));
+      drupal_set_message(\Drupal::translation()->formatPlural(count($ids), 'Updated URL alias for 1 entity.', 'Updated URL aliases for @count entities.'));
```

Comment:
We shouldn't hardcode entity either.. the problem is that there is no generic plural entity label. You could try to rewrite it into something like "for type %label" or so...